tools/devenv/config/test.yml

```
@@ -1,5 +1,5 @@
 setup:
-  codecov_url: https://codecov.io
+  codecov_url: http://localhost:3000
```

why this change? I'd imagine it doesn't really matter but just curious

---

Oh I think it's the thing in `apps/codecov-api/codecov_auth/tests/unit/test_middleware.py`? Leave it if you want, but nit is maybe not using 3000 as we don't really use it anymore for gazebo.

---

Now that the dev settings use `codecov_url` to override `codecov_dashboard_url` instead of ignoring it, running the tests locally was failing since some of the API tests expect the URL to be `http://localhost:3000`

---

Pushed a change to modify the tests instead